frontend/src/components/Home.vue

```diff
@@ -1,135 +1,172 @@
-          <div class="column">
-            <div class="card">
-              <div class="columns is-mobile is-multiline my-0 ml-0">
-                <div id="menu-desktop"
-                     class="column is-narrow has-background-primary has-text-weight-bold p-0">
-                  <aside class="menu">
-                    <ul class="menu-list is-unselectable">
-                      <li v-for="menuItem in menu" :key="menuItem.title">
-                        <a :class="[ {'is-active' : menuItem.title === currentMenu.title}]"
-                           class="pl-2"
-                           @click="currentMenu = menuItem">
-                          <span class="icon is-medium">
-                            <i :class="`fa fa-${menuItem.icon}`"></i>
-                          </span>
-                          {{ menuItem.title }}
-                        </a>
-                      </li>
-                    </ul>
-                  </aside>
+
+    <section id="home">
+
+      <div class="py-6">
+        <div class="container px-6">
+          <div class="columns is-multiline is-mobile is-variable is-8-tablet is-centered is-vcentered">
+            <div v-for="item in tools" :key="item.title" class="column is-3-desktop is-6-mobile is-size-5">
+              <router-link :to="item.route">
+                <div class="card hoverable">
+                  <div class="card-image">
+                    <figure class="image is-4by3">
+                      <img :src="item.img" :alt="item.title" />
+                    </figure>
+                  </div>
+                  <footer class="card-footer has-text-centered">
+                    <a class="card-footer-item has-text-weight-bold is-size-4 is-size-5-mobile">
+                      {{ item.cardLink }}
+                    </a>
+                  </footer>
                 </div>
-                <div class="column more-padding is-v-aligned">
-                  <p class="has-text-justified" v-html="currentMenu.text"></p>
+              </router-link>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="has-background-white-ter py-6">
+        <div class="container px-6">
+          <p class="title is-size-4 has-text-centered pt-4">
+            <router-link :to="news.route" class="has-text-primary">
+              <span class="icon is-large fa-lg">
+                <i :class="`fa fa-${news.icon}`"></i>
+              </span>
+              {{ news. title }}
+            </router-link>
+          </p>
+          <div class="columns is-variable is-8 pt-3">
+            <div class="column is-6 is-size-5">
+              <p>The main versions of <b>Metabolic Atlas</b> are associated with scientific articles as follows.</p><br>
+              <div class="box">
+                <p>Version 2.0:
+                  <i>Genome-scale reconstruction of metabolic networks of model animals
+                     represents a platform for translational research</i>.
```

Comment:
```suggestion
                    represents a platform for translational research</i>.
```

Linter warning about indentation here.